### ISSUE TITLE
Implement submission timer code

### ISSUE THREAD

**Issue by arjun921** [sig-ml/crossval#6]
Prevent users from submitting post contest end date

**Comment by theSage21** [comment]
Yes, this is there in the TODO. I'm putting this off till I'm a little more free. You are welcome to submit a pull request though.
